controller/instance_manager_controller.go

```diff
@@ -1084,17 +1088,17 @@ func (imc *InstanceManagerController) createGenericManagerPodSpec(im *longhorn.I
 	return podSpec, nil
 }
 
-func (imc *InstanceManagerController) createEngineManagerPodSpec(im *longhorn.InstanceManager, tolerations []v1.Toleration, registrySecret string, nodeSelector map[string]string) (*v1.Pod, error) {
+func (imc *InstanceManagerController) createInstanceManagerPodSpec(im *longhorn.InstanceManager, tolerations []v1.Toleration, registrySecret string, nodeSelector map[string]string) (*v1.Pod, error) {
 	podSpec, err := imc.createGenericManagerPodSpec(im, tolerations, registrySecret, nodeSelector)
 	if err != nil {
 		return nil, err
 	}
 
 	secretIsOptional := true
-	podSpec.ObjectMeta.Labels = types.GetInstanceManagerLabels(imc.controllerID, im.Spec.Image, longhorn.InstanceManagerTypeEngine)
-	podSpec.Spec.Containers[0].Name = "engine-manager"
+	podSpec.ObjectMeta.Labels = types.GetInstanceManagerLabels(imc.controllerID, im.Spec.Image, longhorn.InstanceManagerTypeAllInOne)
+	podSpec.Spec.Containers[0].Name = "instance-manager"
 	podSpec.Spec.Containers[0].Args = []string{
-		"engine-manager", "--debug", "daemon", "--listen", "0.0.0.0:8500",
+		"instance-manager", "--debug", "daemon", "--listen", "0.0.0.0:8500",
 	}
```

Comment:
Should we handle the compatibility of the existing and old instance-manager-r/e?

Reply:
Currently, we don't handle recreating previous versions of instance-manager pods. If the old instance-manager-e/r is deleted, it will be recreated with the default instance-manager and in its pod. Any concerns?

Reply:
I see. Then, i think it is ok. Thank you.